include/Basis/SingleParticleBasis.hpp

```diff
@@ -330,7 +330,7 @@ class SingleParticleBasis {
      *
      *  @return the Mulliken operator for a set of AOs
      *
-     *  Note that this method is only available for real matrix representations
+     *  Note that this method is only available for real SQoperators
      */
     template<typename Z = TransformationScalar>
     enable_if_t<std::is_same<Z, double>::value, ScalarSQOneElectronOperator<double>> calculateMullikenOperator(const Vectoru& ao_list) const {
```

Comment:
`ao_list` is a raw vector, whose indices don't really mean anything without a spinor basis. It might be better to wrap this into a class with a specific meaning. Could you open an issue for this and provide your initial thoughts?

Is it possible to generalize this Mulliken partitioning for any first-quantized operator? "Mulliken population" seems to be related to the Mulliken partitioning of the number operator, and it seems your implementation of "atomic S_z" is related to the Mulliken partitioning of the S_z operator, ... What are the similarities in the code if we would like to use another partitioning scheme? Could you open an issue and provide initial thoughts?